### PR TITLE
Update to 1.4.4

### PR DESCRIPTION
**Pull request by djechelon**
Update to latest version of Angular

### CODE REVIEW

**Comment by jamesward** [comment]
Can you also update the `SNAPSHOT` version?

**Comment by djechelon** [comment]
Yea, you probably meant this:

    -    <version>1.3.17-2-SNAPSHOT</version>
    +    <version>1.4.4-1-SNAPSHOT</version>

File: pom.xml [hunk 1]
```diff
@@ -11,14 +11,14 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>angularjs</artifactId>
-    <version>1.3.17-2-SNAPSHOT</version>
+    <version>1.4.4-1-SNAPSHOT</version>
```
Comment (jamesward) [inline pom.xml]: Typically we don't amend the `-1` until we need a new version of the upstream.  So this should be `1.4.4-SNAPSHOT`.  Sorry for the hassle.

**Comment by djechelon** [comment]
Actually, I was thinking about setting up some kind of hook (can Travis to that?) on Github-based webjars, I think most of them, so that when a new version is released its corresponding webjar package is updated and built.

This will save you lots of time and PRs. I am not actually a sponsor of Angular but I need to update the webjar to the latest (thought I urged only to upgrade Smart Table)

**Comment by jamesward** [comment]
We've [discussed auto-updating in the past](https://github.com/webjars/webjars/issues/289) but invested instead in user-driven, on-demand deployments of Bower and NPM WebJars.  The time investment in updating the Classic WebJars has really gone down since the Bower and NPM WebJar stuff was released.  So I'm not sure if we should revisit this.

**Pull request merged**